### ISSUE TITLE
selected line in location list with solarized dark is unreadable

### ISSUE THREAD

**Issue by oryband** [chriskempson/base16-vim#67]
Totally unreadable, check the screenshot.
![screenshot - 03312015 - 11 02 11 am](https://cloud.githubusercontent.com/assets/115387/6914661/7c8b61d4-d795-11e4-98ae-21b16e41a1c3.png)

**Comment by oryband** [comment]
anything new with this?

**Comment by chriskempson** [comment]
Would you mind providing a few steps to replicate this? I'm not familiar with the location list. Thanks.

**Comment by oryband** [comment]
just `:lopen` and you'll see the list. it's a special window that is populated with line locations in multiple files. it's used by [Syntastic](https://github.com/scrooloose/syntastic) (for example) to show syntax errors.

The highlight color needs to be changed to something with a more comfortable contrast

**Comment by oryband** [comment]
http://stackoverflow.com/questions/20933836/what-is-the-difference-between-location-list-and-quickfix-list-in-vim

**Comment by chriskempson** [comment]
![image](https://cloud.githubusercontent.com/assets/473831/16417820/456c387a-3d3f-11e6-9006-e88fff06f78b.png)
Looks okay to me?

**Comment by oryband** [comment]
@chriskempson You are not using solarized

**Comment by adamyonk** [comment]
I've noticed this as well. Happens in the light version, too.

![screen shot 2016-06-28 at 12 54 01](https://cloud.githubusercontent.com/assets/33258/16426615/70b65c3c-3d2f-11e6-8d75-0e5d448e999a.png)

**Comment by chriskempson** [comment]
@oryband both themes map to the same colours do it should affect both. What does that yellow/orange line mean anyway?

**Comment by oryband** [comment]
It's the current selected entry in the list. Pressing enter opens a new
buffer with the file and cursor on the line pointed by that entry.

On Tue, Jun 28, 2016, 21:25 Chris Kempson <notifications@github.com> wrote:

> @oryband <https://github.com/oryband> both themes map to the same colours
> do it should affect both. What does that yellow/orange line mean anyway?
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/chriskempson/base16-vim/issues/67#issuecomment-229138133>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHCu5P65oryBZXdvaPGiPAzJNQv-FiKks5qQWblgaJpZM4D3mMd>
> .
>
-- 
Ory Band - אורי בנד

oryband@gmail.com
OryBand.com <http://oryband.com>

**Comment by oryband** [comment]
You can select other lines - in the example above selecting line 3 and pressing enter will take you that file/line location in a new buffer.

**Comment by chriskempson** [comment]
Got it! Thanks. I've darkened the search text highlighting by two shades. I would make it darker but this colour is used by column highlighting (see picture).

![image](https://cloud.githubusercontent.com/assets/473831/16447506/e2f29b66-3de4-11e6-918a-98e0da46fe2e.png)

**Comment by oryband** [comment]
@chriskempson 
 1. it looks like you changed this for all themes, yes?
 1. can you show an example with solarized-dark and eighties-dark please?

**Comment by chriskempson** [comment]
Yes, it is changed for all themes.

![image](https://cloud.githubusercontent.com/assets/473831/16450165/7e16517e-3df3-11e6-838b-93203b84c166.png)

![image](https://cloud.githubusercontent.com/assets/473831/16450203/cdacf3d2-3df3-11e6-97d8-dc25340afb2b.png)

Seems like I made it darker but killed off the text in the column highlight, might have to make it one shade lighter to avoid that. Should still be readable however 😄

**Comment by oryband** [comment]
@chriskempson you can just make it the same color as visual selection, which for eighties-dark is a light grey

**Comment by oryband** [comment]
@chriskempson also thank you very much for the effort! :+1:

**Comment by oryband** [comment]
@chriskempson ping?

**Comment by chriskempson** [comment]
@oryband Sorry, just got back from a holiday. Are you suggesting for search results I don't change the background colour to yellow but just use the selection colours?

**Comment by oryband** [comment]
Yes, that's sounds the most logical choice IMO. The location list is
practically a list that you open in order to choose one of the lines
presented to you. I think it'll also be more readable.

On Sat, Jul 9, 2016, 01:34 Chris Kempson <notifications@github.com> wrote:

> @oryband <https://github.com/oryband> Sorry, just got back from a
> holiday. Are you suggesting for search results I don't change the
> background colour to yellow but just use the selection colours?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/chriskempson/base16-vim/issues/67#issuecomment-231488798>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHCu52PlQ4s_8nkFOHqM_mDjUZIoonXks5qTtCFgaJpZM4D3mMd>
> .
>
-- 
Ory Band - אורי בנד

oryband@gmail.com
OryBand.com <http://oryband.com>

**Comment by chriskempson** [comment]
I'm sure that will look fine on the location list but it will make any searches made much harder to see. I'm pretty sure that more people search than use the location list so I have to give that preference. The issue here is that I want to be able to set different colours for the location list and search highlighting but as far as I'm aware Vim wont allow me to do this 😦

**Comment by oryband** [comment]
> I'm sure that will look fine on the location list but it will make any searches made much harder to see.

Why? Visual selection color, which is light grey when using a dark colorscheme, is very visible as a highlight. In addition it is much less "noisy" on the screen than a bright yellow, which doesn't contrast well with the dark font color, making the text hard to read.

It's also separated from the main buffer by a big margin line so it will still be visible.

Maybe give this another thought?

**Comment by chriskempson** [comment]
If I change this in Vim the change has to be made in other editor templates too. This is no longer a Vim only issue but perhaps one that should be raised in https://github.com/chriskempson/base16 and it would require a change to the styling guidelines.

**Comment by oryband** [comment]
Ah, I see.

Well, here it goes: Maybe consider this a proposal to the styling guidelines? :shipit:

**Comment by oryband** [comment]
@chriskempson ping?

**Comment by chriskempson** [comment]
@oryband Agreed, feel free to submit such a proposal at https://github.com/chriskempson/base16

**Comment by tony** [comment]
I'm still getting this issue.

a86dfe4   / https://github.com/tony/vim-config-framework/tree/6ab400b5454a7ef4e337a5fe2782675eaebc92ba

<img width="1261" alt="screen shot 2017-06-28 at 9 01 26 pm" src="https://user-images.githubusercontent.com/26336/27668169-0b0a624e-5c45-11e7-9c0f-3844b57bcfaf.png">

Anything that we could paste into our configs to fix it?

**Comment by tony** [comment]
I get similar situation when highlighting: 

<img width="611" alt="screen shot 2017-06-28 at 9 04 56 pm" src="https://user-images.githubusercontent.com/26336/27668238-78b6483a-5c45-11e7-8fa4-f4c32a34471b.png">

**Comment by tbnorth** [comment]
For anyone still experiencing this problem and just looking for a solution ;-)
```
hi QuickFixLine ctermbg=234
```
fixed it for me.
